system/Validation/Validation.php

```diff
@@ -108,6 +108,12 @@ public function __construct($config, RendererInterface $view)
      */
     public function run(?array $data = null, ?string $group = null, ?string $dbGroup = null): bool
     {
+        // If there are still validation errors for redirect_with_input request, remove them.
+        // See `getErrors()` method.
+        if (isset($_SESSION, $_SESSION['_ci_validation_errors'])) {
+            unset($_SESSION['_ci_validation_errors']);
+        }
```

Comment:
`unset()` does not care about existence, why not just call it?
```suggestion
        unset($_SESSION['_ci_validation_errors']);
```

Reply:
@MGatner No, `unset()` care about existence.
See https://github.com/codeigniter4/CodeIgniter4/pull/5844#discussion_r840335862

Reply:
Ah, thank you - sorry I missed that originally. Mobile doesn't always show inline comments for resolved threads.